Source/ChocolateyGui.Common.Windows/Views/RemoteSourceView.xaml

```diff
@@ -310,11 +310,19 @@
                     Style="{DynamicResource PaginationButtonStyle}"/>
         </Grid>
 
-        <ListView Grid.Row="1" x:Name="Packages" ItemsSource="{Binding Packages, Mode=OneWay}"
-                  VerticalAlignment="Stretch"
-                  HorizontalContentAlignment="Stretch"
-                  ScrollViewer.HorizontalScrollBarVisibility="Disabled"
-                  ScrollViewer.IsDeferredScrollingEnabled="False"
-                  MouseDoubleClick="Packages_OnMouseDoubleClick" />
+        <DockPanel Grid.Row="1" LastChildFill="true">
+            <ListView x:Name="Packages" ItemsSource="{Binding Packages, Mode=OneWay}"
+                      VerticalAlignment="Stretch"
+                      HorizontalContentAlignment="Stretch"
+                      ScrollViewer.HorizontalScrollBarVisibility="Disabled"
+                      ScrollViewer.IsDeferredScrollingEnabled="False"
+                      MouseDoubleClick="Packages_OnMouseDoubleClick" />
+            <TextBlock Visibility="{Binding ShowShouldPreventPreloadMessage, Converter={StaticResource BoolToVis}}"
+                       Text="Prevent Preload Feature is enable.  Please search for a package using the textbox above"
+                       HorizontalAlignment="Center"
+                       VerticalAlignment="Center"
+                       FontSize="50"
+                       TextWrapping="Wrap"/>
+        </DockPanel>

```

Comment:
@gep13 This may look better...

```suggestion
<Grid Grid.Row="1">
            <ListView x:Name="Packages" ItemsSource="{Binding Packages, Mode=OneWay}"
                      VerticalAlignment="Stretch"
                      HorizontalContentAlignment="Stretch"
                      ScrollViewer.HorizontalScrollBarVisibility="Disabled"
                      ScrollViewer.IsDeferredScrollingEnabled="False"
                      MouseDoubleClick="Packages_OnMouseDoubleClick" />

            <StackPanel Orientation="Vertical" HorizontalAlignment="Center" VerticalAlignment="Center"
                        Visibility="{Binding ShowShouldPreventPreloadMessage, Converter={StaticResource BoolToVis}}">
                <iconPacks:PackIconOcticons Kind="Search" Width="72" Height="72" Foreground="{DynamicResource MahApps.Brushes.AccentBase}" HorizontalAlignment="Center" Margin="20" />
                <TextBlock Text="No results yet..."
                           HorizontalAlignment="Center"
                           FontSize="28"
                           Margin="10"
                           TextWrapping="Wrap"/>
                <TextBlock Text="Because the Prevent Preload Feature is enabled."
                           HorizontalAlignment="Center"
                           FontSize="22"
                           Margin="5"
                           TextWrapping="Wrap"/>
                <TextBlock Text="Please search for a package using the textbox above."
                           HorizontalAlignment="Center"
                           FontSize="20"
                           Margin="5"
                           TextWrapping="Wrap"/>
            </StackPanel>
        </Grid>
```
![image](https://user-images.githubusercontent.com/658431/78408749-3c32ce80-7608-11ea-8643-1494c77ab3f4.png)